src/compiler.js

```diff
@@ -511,7 +511,7 @@ var Compiler = Object.extend({
             var name = target.value;
             var id = frame.lookup(name);
 
-            if (id == null) {
+            if (id === null || id === undefined) {
```

Comment:
I'm actually open to still using `==` for checking null/undefined. It gets annoying to always have to check for both, and once you learn that `==` matches both, I don't think its too confusing. That's the only case I'd want to use `==`. What do you think (others who see this, feel free to comment too)?